scripts/buildRpm.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
```

Comment:
OK. I will take this and make it similar to our other buildRPM scripts in the next PR